### DISCUSSION TITLE
Exploring

### DISCUSSION THREAD

**Discussion by sunflashke** [adempiere/adempiere#4149] [ADempiere's New Launching]
I am in the  process of identifying an ERP and i am loving the Adempiere... How can i set it up?

**Comment by piracio** [comment]
Hello, 

If you like to install the version 393 with java 8? or the last version 394 with java11?
For the last version. ( I like Linux I will assume the OS is Linux ), rpm base version.

The page https://github.com/adempiere/adempiere/releases have the file  Adempiere_394LTS.tar.gz 
Please download that file. 

The Java can be downloaded from Oracle ( I like the Oracle version ), the file is named jdk-11.0.19_linux-x64_bin.rpm
Depending of the app server you can choose from Tomcat, Jetty or Wildfly, the link https://github.com/adempiere/adempiere/releases have more information.

The database can be 14 or 15 for postgresql, Adempiere supports Oracle too (I am the Oracle DBA :D).

you need to create a user adempiere.
Ex: after installing the postgresql database.
createuser adempiere -s -P #(this will create the user adempiere as super user, this is needed to load a few operators )
createdb adempiere -O adempiere #(This will create a database and the owner will be adempiere)

This may be a good starting point, if you have problems from here, let me know. 
perhaps I may create EZ way to install AD, if you like to test the AD please check the following url https://adqa.prolinux.cl/admin/

Please keep on mind, this adqa is for testing and we may reinstall this site at any given time.

Regards, 
Horacio Miranda.
